<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FtpSelectionDialog</class>
 <widget class="QDialog" name="FtpSelectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>421</width>
     <height>541</height>
    </rect>
   </property>
   <property name="title">
    <string>FTP Server informations</string>
   </property>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>401</width>
      <height>381</height>
     </rect>
    </property>
    <property name="title">
     <string>Content</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Please select the file</string>
     </property>
    </widget>
    <widget class="QTreeWidget" name="fileList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>381</width>
       <height>311</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>401</width>
      <height>33</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QPushButton" name="doneButton">
       <property name="text">
        <string>Done</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>411</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Ftp server</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="url"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="cdToParrent">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="ftp_selection.qrc">
         <normaloff>:/translationManager/images/cdtoparent.png</normaloff>:/translationManager/images/cdtoparent.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QPushButton" name="connectButton">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="ftp_selection.qrc"/>
 </resources>
 <connections/>
</ui>
